[2022-11-10T16:21:16.483+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: a_fourth_dug.fill_dir manual__2022-11-10T16:21:12.653389+00:00 [queued]>
[2022-11-10T16:21:16.490+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: a_fourth_dug.fill_dir manual__2022-11-10T16:21:12.653389+00:00 [queued]>
[2022-11-10T16:21:16.491+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-10T16:21:16.492+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-10T16:21:16.493+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-10T16:21:16.501+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): fill_dir> on 2022-11-10 16:21:12.653389+00:00
[2022-11-10T16:21:16.517+0000] {standard_task_runner.py:55} INFO - Started process 5299 to run task
[2022-11-10T16:21:16.521+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'a_fourth_dug', 'fill_dir', 'manual__2022-11-10T16:21:12.653389+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dug.py', '--cfg-path', '/tmp/tmpn9fpcb66']
[2022-11-10T16:21:16.523+0000] {standard_task_runner.py:83} INFO - Job 27: Subtask fill_dir
[2022-11-10T16:21:16.586+0000] {task_command.py:376} INFO - Running <TaskInstance: a_fourth_dug.fill_dir manual__2022-11-10T16:21:12.653389+00:00 [running]> on host 0b03955da713
[2022-11-10T16:21:16.641+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=eugeny
AIRFLOW_CTX_DAG_ID=a_fourth_dug
AIRFLOW_CTX_TASK_ID=fill_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-11-10T16:21:12.653389+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-10T16:21:12.653389+00:00
[2022-11-10T16:21:16.643+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-10T16:21:16.652+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo 333 > test_directory']
[2022-11-10T16:21:16.731+0000] {subprocess.py:86} INFO - Output:
[2022-11-10T16:21:16.746+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-10T16:21:16.814+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=a_fourth_dug, task_id=fill_dir, execution_date=20221110T162112, start_date=20221110T162116, end_date=20221110T162116
[2022-11-10T16:21:16.854+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-10T16:21:16.889+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
